nix/test/disconnect/disconnect.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs, lib, ... }:
+let
+  targetIp = "192.168.0.1";
+  initiatorIp = "192.168.0.2";
+  common = import ../common.nix { inherit pkgs; };
+in
+{
+  name = "fio_against_nvmf_target";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ gila ];
```

Comment:
should the maintainer name be you?

Reply:
can we remove those entirely? we should all be the maintainers

Reply:
Ok, I'll remove that.